### PR TITLE
CC-1299: Refactor this code to use the AVRO_CODEC config key from storage-common

### PR DESCRIPTION
**Pull request by wicknicks**
Signed-off-by: Arjun Satish <arjun@confluent.io>

### CODE REVIEW

File: kafka-connect-s3/src/main/java/io/confluent/connect/s3/S3SinkConnectorConfig.java [hunk 1]
```diff
@@ -133,7 +133,8 @@
   }
 
   public static ConfigDef newConfigDef() {
-    ConfigDef configDef = StorageSinkConnectorConfig.newConfigDef(FORMAT_CLASS_RECOMMENDER);
+    ConfigDef configDef = StorageSinkConnectorConfig.newConfigDef(FORMAT_CLASS_RECOMMENDER,
```
Comment (kkonstantine) [inline kafka-connect-s3/src/main/java/io/confluent/connect/s3/S3SinkConnectorConfig.java]: Multi-line calls need each argument in its own line, as: 
```java
ConfigDef configDef = StorageSinkConnectorConfig.newConfigDef(
   FORMAT_CLASS_RECOMMENDER,
   AVRO_COMPRESSION_RECOMMENDER
);
```

Comment (wicknicks) [reply kafka-connect-s3/src/main/java/io/confluent/connect/s3/S3SinkConnectorConfig.java]: Thanks, @kkonstantine. Fixed this.

**Comment by wicknicks** [comment]
retest this please

**Comment by wicknicks** [comment]
retest this please

**Pull request merged**